src/pages/index.js

```diff
@@ -165,22 +289,29 @@ profileCloseBtn.addEventListener("click", () => {
 
 editFormElement.addEventListener("submit", handleEditFormSubmit);
 cardForm.addEventListener("submit", handleAddCardSubmit);
-
-initialCards.forEach((item) => { // first parem is the object, second can reference the index of that object and third parem references the intial array
-    const cardElement = getCardElement(item);
-    cardsList.prepend(cardElement);
-});
+avatarForm.addEventListener("submit", handleAvatarSubmit);
 
 addCardNewPost.addEventListener("click", (evt) => {
     evt.stopPropagation();
-    console.log("click");
     openModal(addCardModal);
 });
 
 addCardCloseBtn.addEventListener("click", () => {
```

Comment:
**Could be improved**

You can make a universal handler for any close buttons.

It will look something like this:

```js
// Find all close buttons
const closeButtons = document.querySelectorAll('.modal__close');

closeButtons.forEach((button) => {
  // Find the closest popup only once
  const popup = button.closest('.modal');
  // Set the listener
  button.addEventListener('click', () => closePopup(popup));
});



```

That’s why we make universal css classes for similar elements: here it’s `modal__close` for any close button.

So now you can add 100 modals: their close buttons will be handled automatically.